### DISCUSSION TITLE
How to unpair a specified client in sunshine

### DISCUSSION THREAD

**Discussion by xd-byte** [LizardByte/community#453] [Sunshine] [answered]
I found the Unpair All button in Sunshine Web UI, but I want to unpair the specified client, what should I do?
<img width="1244" alt="unpair" src="https://github.com/LizardByte/Sunshine/assets/53884905/b13109d4-8f49-4983-81a3-9c8011dfb698">

**Comment by gschintgen** [comment, marked as answer]
Such a feature is not yet implemented. But it's being worked on.

**Reply by ReenigneArcher** [reply]
Available now in beta.
